ecosystem/indexer-grpc/indexer-transaction-generator/README.md

```diff
@@ -18,4 +19,21 @@ import_config:
     # A map from versions to dump and their output names.
     versions_to_import:
       123: testnet_v1.json
+# Config to generate the transactions via localnode.
+script_transaction_generator_config:
```

Comment:
nit: can we give a placeholder config file?

Reply:
done